test/ws.js

```diff
@@ -145,6 +160,8 @@ test('ws socket bus', function (t) {
 
         // Tally messages from other buses.
         bus.on('data', function (msg) {
+          watchdog.reset();
```

Comment:
for this library, `reset` is akin to petting the watchdog.  It does reset the internal timer.  If the timer ever falls to 0, :boom: